frontend/server/src/main/java/com/amazonaws/ml/mms/util/ConfigManager.java

```diff
@@ -100,8 +106,43 @@ public boolean isDebug() {
                 || Boolean.parseBoolean(prop.getProperty(DEBUG, "false"));
     }
 
-    public int getPort() {
-        return getIntProperty(PORT, isUseSsl() ? 8443 : 8080);
+    public URI getAddressProperty(String key, String host, int port)
```

Comment:
This value should support following format:
http(s)://127.0.0.1:8443
http(s)://[::1]:8443
/tmp/.netty_sock

For now, we only implement http/https.

So you can remove use_ssl. and management/inference can have different ssl settings.
But we'd like return SocketAddress object, it can be a UDS in future.